templates/pages/category.html

```diff
@@ -24,11 +24,13 @@ <h1 class="page-heading">{{category.name}}</h1>
 {{{category.description}}}
 {{{snippet 'categories'}}}
 <div class="page">
-    {{#or category.subcategories category.faceted_search_enabled category.shop_by_price}}
-        <aside class="page-sidebar" id="faceted-search-container">
-            {{> components/category/sidebar}}
-        </aside>
-    {{/or}}
+    {{#if theme_settings.shop_by_price_visibility}}
```

Comment:
This won't work. Currently it is just shop by price but if you have faceted search enabled there are other elements in the side bar https://github.com/bigcommerce/cornerstone/blob/67305e81473f01a93a351ad4b2e80b60fc9c66ba/templates/components/category/sidebar.html. 

This change will hide those blocks as well.

Reply:
figured as such - fixed